### PR TITLE
chore: dependabot disable groups.

### PR DESCRIPTION
**Pull request by aviv1620**
# Description
If all the update grouped to one PR we don't know witch update breaking changes.
if we not set up the `open-pull-requests-limit` the [default it only five PRs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit),

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -11,8 +11,4 @@ updates:
       interval: "weekly"
     reviewers:
       - "aviv1620"
-    groups:
-      production-dependencies:
-        dependency-type: "production"
-      development-dependencies:
-        dependency-type: "development"
+    open-pull-requests-limit: 100
```
Comment (NoamGaash) [inline .github/dependabot.yml:14]: That's a great idea, but don't think we have the capacity to manage 100 open pull request. Don't you think that 5 will be enough?
You can run npm audit and such locally to find things that should be upgraded

Comment (aviv1620) [reply .github/dependabot.yml:14]: The tests take very long time to complete.
And we have 42 dependencies updates.
(23 development-dependencies + 18 production-dependencies)

So I prefer at least capacity of 50 PRs to check all the updates with the same day.

Screenshot 2024-03-20 174616

Comment (NoamGaash) [reply .github/dependabot.yml:14]: tests are running in parallel,
I don't see how anyone will go through 100 pull requests

Comment (aviv1620) [reply .github/dependabot.yml:14]: OK.
So how many PRs we want?

Comment (NoamGaash) [reply .github/dependabot.yml:14]: I think 5 if optimal, but feel free to set it to any number that you think you'll be able to maintain
If we'll see we get too many pull requests that no-one is taking care of, I'll revert it back to 1.

**Review by NoamGaash — APPROVED** [review]

**Pull request merged**